pupil-api/src/services/jwt.service.ts

```diff
@@ -1,82 +0,0 @@
-'use strict'
```

Comment:
My understanding is that this endpoint is not responsible for checking the pupil’s JWT token validity anymore, as part of the table storage changes.

Reply:
That's correct.  The SQL database is replaced by a high-performance data-store.